src/WebJobs.Extensions.ApiHub/Table/TableClientValueProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub.Table
+{
+    internal class TableClientValueProvider : IValueProvider
```

Comment:
even for nested classes? It works out pretty nicely in that case. 

---
In reply to: [64083666](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/98#discussion_r64083666) [](ancestors = 64083666)